zkevm-circuits/src/evm_circuit/execution/codecopy.rs

```diff
@@ -49,14 +49,15 @@ impl<F: Field> ExecutionGadget<F> for CodeCopyGadget<F> {
     fn configure(cb: &mut ConstraintBuilder<F>) -> Self {
         let opcode = cb.query_cell();
 
-        // Query elements to be popped from the stack.
-        let dst_memory_offset = cb.query_cell_phase2();
-        let code_offset = cb.query_word_rlc();
+        let code_size = cb.query_cell();
+
         let size = cb.query_word_rlc();
+        let dst_memory_offset = cb.query_cell_phase2();
+        let code_offset = WordByteCapGadget::construct(cb, code_size.expr());
```

Comment:
We're using `WordByteCapGadget` for `code_offset`, but I don't see any place where we check `code_offset.overflow` or `code_offset.within_range`.  So the internal `IsZero` gadget seems to be unused?